Fulfillment/Troubleshooting/01_HandleErrorStatus.md

```diff
@@ -0,0 +1,91 @@
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Fulfillment > Dispatch notes > Tab LIST*
+
+![Error status](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNotesErrorStatus.png "[Error status]")
+
+1. Click the dispatch note displaying an error in the *Status* or the *Shipment status* columns.  
+    The *Dispatch note "Dispatch note ID"* view is displayed. 
+
+2. Click the *Logs* sub-tab.  
+    The list of intents related to the selected dispatch note are displayed.
+
+    ![Logs](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNoteLogsError.png "[Logs]")
+
+3. Click the desired intent from the list to access further information.  
+    The list of messages is displayed.  
+
+    ![Messages](../../Assets/Screenshots/Fulfillment/DispatchNotes/DispatchNoteLogsMessages.png "[Messages]")
+
+4. If necessary, click the relevant message for further information.  
+    The *Detail of log message "Log message title"* is displayed.
+ 
+5. If no further information is detailed or the information displayed is not conclusive, click the [CLOSE] button in the upper right corner to close the current view.  
+    The list of dispatch notes is displayed again. 
+
+6. Copy the dispatch note number in the *Document number* column to your clipboard.
+
+    > [Note] The document number equals the delivery note number in the *Order management* module.
+
+7. Switch to the *Workflows* module and select the *Processes* menu entry.  
+    The list of processes is displayed. 
+
+    ![Processes](../../Assets/Screenshots/ActindoWorkFlow/Processes/FaultyProcess.png "[Processes]")
+
+8. Paste the document number in the search field and press the **ENTER** key to start the search.  
+    The process related to the selected dispatch note number is displayed. 
+
+9. Click the process to display further process details.  
+    The *Process ID: "Process ID", Workflow: "Workflow name"* view is displayed. 
+
+    ![Process ID](../../Assets/Screenshots/ActindoWorkFlow/Processes/FaultyProcessID.png "[ProcessID]")
+        
+    > [Info] The action where the error has ocurred is displayed in red, both in the process diagram as well as in the process actions listed below.
+
+10. Click the *Logs* tab to display all process-related logs.  
+    The list of logs is displayed.
+
+     ![Logs](../../Assets/Screenshots/ActindoWorkFlow/Processes/Logs.png "[Logs]")
+
+    > [Info] If necessary, the logs can be filtered by log level.
+
+11. Click the log displaying the error. 
+    The *Log ID "Log ID"* view is displayed.
+
+    ![Log ID](../../Assets/Screenshots/ActindoWorkFlow/Processes/LogID.png "[LogID]")
+
+12. Check the log message for further information about the error. Once the error cause has been established, an appropriate solution can be implemented.  
+
+    > [Info] Depending on the source of the error, it may be necessary to take action in one or more additional modules or even to contact externally the fulfillment partner. 
```

Comment:
```suggestion
    > [Info] Depending on the error cause, it may be necessary to take action in one or more additional modules or even to contact the fulfillment partner. 
```
Brauche ich die Info überhaupt?